test/request.js

```diff
@@ -158,11 +158,26 @@ describe('swagger request functions', function () {
 
     expect(req.method).toBe('DELETE');
     expect(req.headers.Accept).toBe('application/json');
-    expect(req.headers['Content-Type']).toBe(undefined);
+    expect(req.headers['Content-Type']).toBe('application/json');
```

Comment:
You should not set `content-type` to `application/json` if there is no body.  That will cause many parsers--including express--to give a JSON parse error